src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -64,7 +64,7 @@ export default function PrivateMenu() {
   const title = `${session?.user?.name}\n(${session?.user?.email})`
 
   const getAvatar = () => {
-    const { data: user } = getCurrentPerson(Boolean(router.query?.register))
+    const { data: user } = useCurrentPerson()
```

Comment:
Hooks are only used inside the top level of components :D.
You would have to call the hooks inside the component and then pass the person to `getAvatar(person: Person)`.

Reply:
Actually since getAvatar is inside the component you wouldn't need to pass it, but you need to hook to be on the topLevel of the component and not be conditionally called. Read https://reactjs.org/docs/hooks-rules.html for more information